examples/kubeflow-mlops/README.md

```diff
@@ -0,0 +1,146 @@
+# Kubeflow on EKS
+The Kubeflow project is dedicated to making deployments of machine learning (ML) workflows on Kubernetes simple, portable and scalable.
+Our goal is not to recreate other services, but to provide a straightforward way to deploy best-of-breed open-source systems for ML to diverse infrastructures.
+Anywhere you are running Kubernetes, you should be able to run Kubeflow.
+
+This example deploys the following resources
+
+- Creates EKS Cluster Control plane with public endpoint (for demo purpose only) with a managed node group
+- Deploys application load balancer and EBS SCI driver
+- Deploy Kubeflow (v1.5.1) on the EKS cluster
+
+Note: we use EKS 1.21 here which is the latest EKS version supported by Kubeflow. see reference below <br>
+https://awslabs.github.io/kubeflow-manifests/docs/about/eks-compatibility/
+
+## Prerequisites:
+
+Ensure that you have installed the following tools on your machine.
+
+1. [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+2. [kubectl](https://Kubernetes.io/docs/tasks/tools/)
+3. [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+4. [Kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/)
+
+
+
+## Deploy EKS Cluster with Kubeflow feature
+
+Clone the repository
+
+```sh
+git clone https://github.com/aws-ia/terraform-aws-eks-blueprints.git
+```
+
+Navigate into one of the example directories and run `terraform init`
+
+```sh
+cd examples/kubeflow-mlops
+terraform init
+```
+
+Run Terraform plan to verify the resources created by this execution.
+
+```sh
+terraform plan
+```
+
+**Deploy the EKS cluster**
+
+```sh
+terraform apply
+```
+
+Enter `yes` to apply.
+
+**Deploy kubeflow**
+* clone kubeflow source code
+```sh
+export KUBEFLOW_RELEASE_VERSION=v1.5.1
+export AWS_RELEASE_VERSION=v1.5.1-aws-b1.0.0
+git clone https://github.com/awslabs/kubeflow-manifests.git && cd kubeflow-manifests
+git checkout ${AWS_RELEASE_VERSION}
+git clone --branch ${KUBEFLOW_RELEASE_VERSION} https://github.com/kubeflow/manifests.git upstream
+```
```

Comment:
It would be nice to have Helm chart to deploy the add-on. If we don't have one then we may have to create one. I can see that Helm chart is missing in kubeflow-aws distro.

Reply:
totally agree. let me sync with kubeflow community to see if it is under their kubeflow 1.6 roadmap. if not, will try to create by ourselves. I am also working with Theofilos Papapanagiotou who recently joined in AWS from kubeflow community. 
Those works may take some time. Will keep you updated. 
Can we use this example as a starting point? would like to have something to demo to our clients when delivering labs.
thanks.